jupyterhub/handlers/base.py

```diff
@@ -416,7 +416,7 @@ async def get_current_user(self):
                 self.log.exception("Error getting current user")
         return self._jupyterhub_user
 
-    def _resolve_scopes(self):
+    def _resolve_roles_and_scopes(self):
         self.raw_scopes = set()
```

Comment:
Maybe we should rename the `raw_scopes` to `expanded_scopes` here too as this is the format the scopes are in here before being parsed.

Reply:
Yeah, that's a good idea to be consistent

Reply:
Agree; there are also some references called `raw_scopes` left in `jupyterhub/roles.py`.

Reply:
Thank you both, coming up.